controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -681,7 +682,25 @@ def test_delete_snapshot_with_stopfcmap_raise_error(self, mock_warning):
         mock_warning.return_value = False
         self.svc.client.svctask.stopfcmap.side_effect = [CLIFailureError('error')]
         with self.assertRaises(CLIFailureError):
-            self.svc.delete_snapshot("test_snapshot")
+            self.svc.delete_snapshot("test_snapshot", "internal_id")
+
+    def _prepare_mocks_for_delete_snapshot_2_0(self):
+        self.svc.client.svctask.addsnapshot = Mock()
+
+    def _test_delete_snapshot_2_0_rmsnapshot_cli_failure_error(self, error_message_id, expected_error):
```

Comment:
```suggestion
    def _test_delete_snapshot_flashcopy_2_0_rmsnapshot_cli_failure_error(self, error_message_id, expected_error):
```
or if we don't want the storage feature name in the test functions name:
```suggestion
    def _test_delete_snapshot_rmsnapshot_cli_failure_error(self, error_message_id, expected_error):
```

Reply:
done